qml/Greeter/ButtonPrompt.qml

```diff
@@ -0,0 +1,59 @@
+import QtQuick 2.4
+import Ubuntu.Components 1.3
+import "../Components"
+
+FocusScope {
+    id: root
+    objectName: "promptButton"
+    focus: true
+
+    property string text
+    property bool isSecret
+    property bool interactive: true
```

Comment:
this can be an alias to the `enabled` property. Then you can remove the `triggered()` function and in the MouseArea do just:
```
    MouseArea {
        anchors.fill: parent
        onClicked: root.clicked();
    }
```

Reply:
Hello and thank you for the suggestions. 
The function triggered() is also called when enter/space is pressed on the keyboard, what should I do with the keybindings?

Reply:
you can just directly call `root.clicked()`, because the key presses will not be received if the item is disabled.